sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -417,6 +417,9 @@ object FunctionRegistry {
     expression[TimeWindow]("window"),
     expression[MakeDate]("make_date"),
     expression[MakeTimestamp]("make_timestamp"),
+    expression[Millennium]("millennium"),
+    expression[Century]("century"),
+    expression[Decade]("decade"),
```

Comment:
If this is not support in PostgreSQL, I'd like to recommend to remove these.

cc @gatorsmile

Reply:
I removed them